packages/jest-fake-timers/src/modernFakeTimers.ts

```diff
@@ -72,12 +84,24 @@ export default class FakeTimers {
     }
   }
 
+  async advanceTimersToNextTimerAsync(): Promise<void> {
```

Comment:
Would it make sense to implement `steps` similar to `advanceTimersToNextTimer(steps?)`? Just wondering if that is possible.

The rest looks clean and nice for my eye. Thanks!